src/screens/SSICredentialsOverviewScreen/index.tsx

```diff
@@ -72,24 +78,39 @@ class SSICredentialsOverviewScreen extends PureComponent<IProps, IState> {
     );
   };
 
-  renderItem = (itemInfo: ListRenderItemInfo<ICredentialSummary>): JSX.Element => (
-    <SSISwipeRowViewItem
-      listIndex={itemInfo.index}
-      viewItem={
-        <SSICredentialViewItem
-          id={itemInfo.item.id}
-          title={itemInfo.item.title}
-          issuer={itemInfo.item.issuer}
-          issueDate={itemInfo.item.issueDate}
-          expirationDate={itemInfo.item.expirationDate}
-          credentialStatus={itemInfo.item.credentialStatus}
-          properties={[]}
+  renderItem = (itemInfo: ListRenderItemInfo<ICredentialSummary>): JSX.Element => {
+    const credentialItem = (
+      <SSICredentialViewItem
+        id={itemInfo.item.id}
+        title={itemInfo.item.title}
+        issuer={itemInfo.item.issuer}
+        issueDate={itemInfo.item.issueDate}
+        expirationDate={itemInfo.item.expirationDate}
+        credentialStatus={itemInfo.item.credentialStatus}
+        properties={[]}
+      />
+    );
+
+    return this.props.activeUser.identifiers.some((identifier: IUserIdentifier) => identifier.did === itemInfo.item.issuer.name)
```

Comment:
We should also check for the credential type. Current code basically means that you cannot delete any credential that is issued by the user. In the future we might add other self-asserted credential use cases, not being the wallet identity credential. So filtering on that makes sense

Reply:
added a check for the type/title of the vc